packages/integrations/cloudflare/src/virtual.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module 'virtual:astro-cloudflare:config' {
+	export const sessionKVBindingName: string;
```

Comment:
You could reuse the type from the vite plugin, eg. something like this:
```ts
export = import('./src/vite-plugin-config.js').CloudflareConfig
```

Reply:
Feel like its a little too cute, just makes it harder to track down.